sql/migrate/migrate.go

```diff
@@ -46,15 +46,29 @@ type (
 		// A Comment describes the change.
 		Comment string
 
-		// Reverse contains the "reversed statement" if
-		// command is reversible.
-		Reverse string
+		// Reverse contains the "reversed" statement(s) if
+		// the command is reversible.
+		Reverse any // string | []string
```

Comment:
Why not `[]string`? What code would we break by using `[]string`?

Reply:
All codebase uses `string` except for one case. I preferred `any` for this reason, but also there's a chance we'll add support for `*Change` in the future (e.g., adding a comment per statement).